src/handlers/createHandler.ts

```diff
@@ -494,7 +494,7 @@ export default function createHandler<
                 handlerTag: this.handlerTag,
               }
             : {}),
-          testID: this.props.testID,
+          ...(this.props.testID ? { testID: this.props.testID } : {}),
```

Comment:
```suggestion
          testID: this.props.testID ?? child.props.testID,
```
HI! Thank you for taking the time to look into this ❤️ . I would really appreciate it if you could change this line to the other one you mentioned in your comment. The `??` operator returns the right value only when the left value is `null` on `undefined`, not any falsy value. While I don't think it will cause problems, I would rather be explicit about this.

Reply:
of course!